site.social.conf.template

```diff
@@ -0,0 +1,130 @@
+limit_req_zone $http_x_forwarded_for zone=sdc:32m rate=100r/m;
+proxy_cache_path /var/cache/nginx levels=1:2 inactive=6h keys_zone=sdc_cache:100m;
+
+server {
+
+    listen 8081;
+    server_name $SERVER_NAME;
+
+    # ddos blacklist
+    deny 122.8.125.184;
+    deny 130.206.99.1;
+    deny 177.226.255.170;
+    deny 179.127.252.154;
+    deny 192.119.168.147;
+    deny 202.55.183.226;
+    deny 202.62.17.217;
+    deny 69.125.96.181;
+    deny 79.173.237.10;
+    deny 83.48.73.254;
+    deny 89.35.44.248;
+    deny 92.114.178.198;
+    deny 99.197.236.213;
+
+    # ignore csp violations
+    location /api/v1/csp_violation {
+        return 403;
```

Comment:
Has this been here a while? Wondering why, it would be useful.

Reply:
yes - it is copied from the existing config, not something new

let's discuss